nginx/conf.d/extra_protect.conf

```diff
@@ -0,0 +1,34 @@
+# deny all internal locations also default phpmyadmin
+location ~ /(app|bin|var|nginx|phpserver|vendor|php[mM]y[aA]dmin|pma)/ { deny all; }
+
+# deny access per default .htaccess rules
+location ~ /media/(customer/|downloadable/|import/|theme_customization/.*\.xml) { deny all; }
+location ~* /(media|static)/.*\.php$ { deny all; }
+location ~ /errors/.*\.xml { deny all; }
+location ~ Gruntfile\.js { deny all; }
+
+## deny cron and files with the obvious names. favorite entry points for hackers and script kiddie
+location ~* (cron|phpminiadmin|pma|sqlyog|adminer.+)\.php { deny all; }
+
+# deny auth and composer
+location ~ (auth|package|composer)\.(json|lock)$ { deny all; }
+
+# deny files that only for internal use (work with ssh or use ftp to download)
+#location ~ /\. { return 444; } # <= block everythig that starts with \.<dot
+location ~ /\.(svn|git|hg|htpasswd|bash|ssh|php_cs) { return 444; }
+location ~ /.*\.(sh|pl|swp|phar|log|ini|sql|conf|yml|zip|tar|.+gz)$ { return 444; }
```

Comment:
actually as i remember we have been working with firewall , like these 403 can be configured with log scanners like fail2ban and csf firewall and block multiple abusers, where 444 the same, the only difference is that 444 will terminate request immediately and user just gets no response back. need to revise this logic.